resources/user-guides/attach-data.md

```diff
@@ -60,20 +60,20 @@ When defining a payload for the attach data action, the payload must match the f
 {% tab title="Analytics" %}
 The following sample shows how to _attach data_ to all outgoing `TrackAction` Analytics network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#analytics-create-rule)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#analytics-select-an-event)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#analytics-define-the-action)
-4. [Save and rebuild the property](attach-data.md#analytics-save-the-rule-and-rebuild-your-property)
+1. [Create a new **Rule**](#analytics-create-rule)
+2. [Select the **Event** you want to trigger the rule](#analytics-select-an-event)
+3. [Select the **Action** to attach data and define your payload](#analytics-define-the-action)
+4. [Save and rebuild the property](#analytics-save-the-rule-and-rebuild-your-property)
```

Comment:
please add a period at the end of each step, like this:
"4. [Save and rebuild the property](#analytics-save-the-rule-and-rebuild-your-property)."